### ISSUE TITLE
Facing Fatal error: Class 'Monolog\Logger' not found issue

### ISSUE THREAD

**Issue by janardhanpasumarthi** [propelorm/Propel#880]
Hello All,

If there are any errors in the propel usage during development, it is trying to log that error in log file. While doing that, it is throwing error - Fatal error: Class 'Monolog\Logger' not found. Provided correct path for log file in logger config.

Can anyone please help me to solve this issue.

Thanks in advance.

**Comment by nibsirahsieu** [comment]
this can be closed, propel1 doesn't use monolog

**Comment by luismldias** [comment]
Hello nibsirahsieu,

Agreed, it can be closed since the issues got solved. However, please note that in composer example above, the propel version is 2.0.

**Comment by nibsirahsieu** [comment]
yep.. i saw that :), but the issue/question should be opened in the propel 2 repo https://github.com/propelorm/Propel2

**Comment by luismldias** [comment]
True :)
